fe/fe-core/src/test/java/com/starrocks/sql/plan/TestCommonTypeInvolvingDecimalTypes.java

```diff
@@ -0,0 +1,71 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+package com.starrocks.sql.plan;
+
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class TestCommonTypeInvolvingDecimalTypes extends PlanTestBase {
```

Comment:
There are many decimal v3 test classes. suggest  move all decimate v3 related tests to one test class.
1. which could reduce UT time
2. which could help use know which point we don't test